### DISCUSSION TITLE
Updated add-on: Pack and Go-Go

### DISCUSSION THREAD

**Discussion by macmarrum** [freeplane/freeplane#1106] [General]
# Pack and Go-Go

_Originally **Pack And Go** by Volker Börchers - no longer maintained_

**Pack and Go-Go** adds a menu item to create a ZIP file with the mindmap, packing also its images and linked mindmaps (without dependencies).

> Note: All files are loaded into RAM. If you get an Out Of Memory Error, see https://docs.freeplane.org/attic/old-mediawiki-content/Troubleshooting.html#freeplane-throws-out-of-memory-exception

## Version Compatibility

Works with Freeplane version 1.9.1 and above.

## Download

[**PackAndGoGo.addon.mm**](https://github.com/macmarrum/freeplane-PackAndGoGo/releases/)

**Comment by beccon4** [comment]
Really nice. I installed it into my new Freeplane 1.11.1 and it does what it should.

Unfortunately  we do not find it on the officional AddOn list yet - which is when you search from Freeplane itself or look here: https://docs.freeplane.org/getting-started/Add-ons_(install).html#-pack--go   There we still have the old outdated version.

Make Things Easier: :-) Perhaps we can convince Upstream to include PackAndGo into the main Freeplane distribution. It would help al lot - not only in my use case: Sending mindmaps with screenshots etc. around.

**Reply by macmarrum** [reply]
Thanks for your feedback.
The [add-ons page](https://docs.freeplane.org/getting-started/Add-ons_(install).html) has been updated to list Pack and Go-Go.
